src/main/scala/zio/redis/api/Geo.scala

```diff
@@ -0,0 +1,84 @@
+package zio.redis.api
+
+import zio.redis.Command
+import zio.redis.Input._
+import zio.redis.Output.{ LongOutput, StreamOutput, StringOutput }
+
+trait Geo {
+  lazy val geoAdd =
+    Command(
+      "GEOADD",
+      Tuple2(
+        StringInput,
+        NonEmptyList(Tuple2(GeoLongLatInput, StringInput))
+      ),
+      LongOutput
+    )
+
+  lazy val geoDist =
+    Command(
+      "GEODIST",
+      Tuple4(
+        StringInput,
+        StringInput,
+        StringInput,
+        OptionalInput(GeoRadiusUnitInput)
+      ),
+      StringOutput
```

Comment:
I'd represent it as double, optional one according to the specs.

Reply:
We were missing both `OptionalOutput` and `DoubleOutput` so I added them, let me know if that is not what you meant.